lms/templates/seq_module.html

```diff
@@ -98,9 +99,15 @@
       </ol>
     </nav>
   </div>
+  % endif
   % if not exclude_units:
   % if gated_content['gated']:
     <%include file="_gated_content.html" args="prereq_url=gated_content['prereq_url'], prereq_section_name=gated_content['prereq_section_name'], gated_section_name=gated_content['gated_section_name']"/>
+  % elif gated_sequence_fragment:
```

Comment:
could a sequence include a timed exam and content outside of the timed exam in the same sequence?